src/layout/progressBar/progressBar.tsx

```diff
@@ -80,16 +84,54 @@ const ProgressBar = () => {
                                 />
                                 {h.type}
                             </p>
-                            <a
-                                className="etherscan"
-                                target="_blank"
-                                // href={`${EXPLORER_URL}/tx/${h.id}`}
-                            >
-                                <span>Etherscan</span>
-                                <img
-                                    src={require('../../../public/images/etherscan-white.svg')}
-                                />
-                            </a>
+                            {h.isSuccess ? (
```

Comment:
When the render conditions are confusing I like to write multiple conditional renders like this:

```
{h.isSuccess && <a />}
{!h.isSuccess && h.type === ActionType.Post && <a />}
{!h.isSuccess && h.type === ActionType.Comment && <a />}
{!h.isSuccess && h.type === ActionType.Vote && h.data !== undefined && <a />}
```

It's a bit more code but it makes things more clear.